packages/orchestration/src/client/api/schema/azure-content-safety-filter-config.ts

```diff
@@ -9,7 +9,7 @@ import type { AzureContentSafety } from './azure-content-safety.js';
  */
 export type AzureContentSafetyFilterConfig = {
   /**
-   * String represents name of the filter provider.
+   * String represents name of the filter provider
```

Comment:
[pp] I think these changes are fine for now. But it's better to have proper linted code. I would like to adjust our eslint config to fix such issues (and only disable consistent-typedefinitions rule).